inference-engine/thirdparty/clDNN/tests/test_cases/pooling_gpu_test.cpp

```diff
@@ -1942,6 +1942,133 @@ TEST(pooling_forward_gpu, b_fs_yx_fsv4)
     } // for (int j = 0; F_array[j]; i++)
 }
 
+TEST(pooling_forward_gpu, b_fs_yx_fsv16)
+{
+    int B_array[] = {  16,    4, 0 };  // Batch
```

Comment:
Those cases can be tested with existing `pooling_random_test`.
This will ensure that results are compared to common cpu reference instead of best gpu implementation for these param.

Reply:
Test removed, and added new case for pooling_random_test so it will test b_fs_yx_fsv16